src/Parallel-Solvers/Parallel-Explicit/SGH_Solver/FEA_Module_SGH.cpp

```diff
@@ -1419,7 +1419,7 @@ void FEA_Module_SGH::setup(){
     //initialize if topology optimization is used
     if(simparam->topology_optimization_on || simparam->shape_optimization_on){
       init_assembly();
-      //assemble_matrix();
+      assemble_matrix();
```

Comment:
Not sure why its trying to reset that particular change, maybe some commits haven't been synced in your branch?

Reply:
Guess so, I noticed that but figured it was something I forgot. Easy to put that comment back on though.

Reply:
if you hit the sync fork button in your forks code viewer while on the branch for this PR (ejl-pvtu) it should address that